oauth2_http/java/com/google/auth/oauth2/ServiceAccountCredentials.java

```diff
@@ -576,36 +592,42 @@ public AccessToken refreshAccessToken() throws IOException {
 
     HttpRequestFactory requestFactory = transportFactory.create().createRequestFactory();
     HttpRequest request = requestFactory.buildPostRequest(new GenericUrl(tokenServerUri), content);
+
+    if (this.defaultRetriesEnabled) {
```

Comment:
Does it make sense to just allow configuring the number of retries and using the default if no configured value is set?

Reply:
As explained in the doc - we don't want to enhance retry functionality. Instead, we want to disable it for as many clients as possible.

Therefore we keep retries and fix them to the spec. But if someone wants custom retry strategy - guidance is to disable library retries and retry on client level. That guidance will be published as AIP, now on review.